api/osrd_infra/schemas/infra.py

```diff
@@ -1,14 +1,17 @@
 from enum import Enum
```

Comment:
The infra generator script needs to be adapted to these changes

Reply:
Done